woocommerce/woocommerce-android
# [Shipping Labels Revamp] Fix custom package dimension units

Why
==========
Every Package request comes with the Store Options data. This data is essential for the entire package selection structure, but the data wasn't being held by the main ViewModel, which blocked some scenarios where Store Options were required after the request.

How
==========
Updates the Package selection flow to keep the StoreOptions data returned by the Packages endpoint, and map the dimensions unit in it to the Custom package creation, allowing the Custom package to keep the correct unit type.

How to Test
==========
1. Simply make sure that all three types of Package selection are working as expected.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
